server.elm

```diff
@@ -1309,6 +1309,10 @@ div#modify-menu {
 .modify-menu-icon:hover {
   background-color: var(--context-button-color-hover);
 }
+/*modify-menu-icon#internalLinkMode {
+  padding-left: 10px;
+  display: table-row;
+}*/
```

Comment:
Can you please remove this comment, or add an explanation why you want to keep it.